cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/MapReduceCounterCollectionService.java

```diff
@@ -64,6 +69,23 @@ protected void publish(Iterator<MetricValue> metrics) throws Exception {
     }
   }
 
+  public static Map<String, String> parseTags(String counterGroupName) {
+    // see publish method for format info
+
+    if (counterGroupName.startsWith("cdap.")) {
```

Comment:
I don't understand this part. If the counter group name starts with cdap, we return an empty map?